nautobot/core/management/commands/post_upgrade.py

```diff
@@ -85,30 +93,36 @@ def handle(self, *args, **options):
 
         # Run trace_paths
         if options.get("trace_paths"):
-            print("Generating cable paths...")
+            self.stdout.write("Generating cable paths...")
             call_command("trace_paths", no_input=True)
             print()
```

Comment:
Should these prints also be changed to self.stdout.write()?

Reply:
Yes.